README.md

```diff
@@ -12,3 +12,59 @@ Almond is part of Open Thing Platform, a research project led by
 prof. Monica Lam, from Stanford University.  You can find more
 information at <https://thingpedia.stanford.edu/about>.
 
+## Installation
+
+This assumes that you have [node](https://github.com/nodejs/node) and [yarn](https://github.com/yarnpkg/yarn) installed, with at least **2 GB** of available diskspace. Please be sure to be on a **fast internet connection** before going through the installation.
+
+1. Clone this repo on your machine.
+```
+git clone https://github.com/stanford-oval/almond-server.git
+```
+
+2. Install libpulse.
+```
+# On Ubuntu/Debian
+sudo apt-get install libpulse-dev
+```
+
+3. Install [mimic](https://github.com/MycroftAI/mimic1).
```

Comment:
This will go away soon and use almond-voice's TTS right?

Reply:
Yes, but I think my plan was to keep the chat in beta (side-by-side with old chat) for a couple of months while users submit feedback/report bugs.

Reply:
Yep